app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/coursedetail/students/view/StudentListFragment.kt

```diff
@@ -59,11 +59,9 @@ class StudentListFragment : Fragment() {
         })
     }
 
-    private fun initializeToaster() {
-        studentsViewModel.showMsg.observe(this, androidx.lifecycle.Observer {
-            val message = it.showMsg
-            context?.showToast(message)
-        })
+    override fun showMessage(message: String) {
+        view?.showSnackbar(message)
+
```

Comment:
don't remove the initialiseToaster method.

Just remove
'context?.showToast(message)'
and instead use
'view?.showSnackbar(message)'

Also, initializeToaster will be renamed to initialiseSnackbar